### PR TITLE
Add continuous deployment action

### PR DESCRIPTION
**Pull request by carpenterd777**
(Must merge #198 before this can be merged) 

Creates a Github Action to automatically deploy updates to the main branch on Expo. All of you should be able to access the Expo dashboard [here](https://expo.dev/accounts/peer-stevens/projects/peer). If @aej11a shares his iOS credentials with the team via Expo, then we can also build for iOS through Expo. Until then, we can only build for Android(on Expo).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,17 +12,10 @@ Run:
 
 ```
 npm i -g yarn
-yarn global add expo-cli
+npm i -g expo-cli
+npm i -g eas-cli
```
Comment (elenirotsides) [inline README.md:16]: What is `eas-cli`?

Comment (carpenterd777) [reply README.md:16]: `eas-cli` is the CLI for [Expo Application Services](https://docs.expo.dev/build/introduction/). It's the service expo provides to build our app for us

Comment (elenirotsides) [reply README.md:16]: Cool, thanks!

**Review by elenirotsides — APPROVED** [review]
OH DANG, I thought I clicked 'Approve' the other day...sorry!

**Pull request merged**